icaruscode/Analysis/crtpmtmatching_parameters.fcl

```diff
@@ -40,7 +40,9 @@ CRTPMTmatchingparams_standard:
   PMTADCThresh:          400  # ADC, this value was suggested from A. Scarpelli to mimic the PMT trigger.
   nOpHitToTrigger:       5  # Number of PMT above threshold to mimic the PMT trigger.
   TimeOfFlightInterval:  100 # ns, time difference between CRT Hit and Optical Flash to confirm the match.
-  GlobalT0Offset:        1.6e6  # ns, (copied from ref: https://github.com/SBNSoftware/icaruscode/blob/v09_37_02_01/icaruscode/CRT/crtsimmodules_icarus.fcl#L11)  
+  #  GlobalT0Offset:        1600043 #1.6e6 + 43 ns   # ns, (copied from ref: https://github.com/SBNSoftware/icaruscode/blob/v09_37_02_01/icaruscode/CRT/crtsimmodules_icarus.fcl#L11)  
+   GlobalT0Offset:        1599957 # 1.6e6 - 43 for 
```

Comment:
This is confusing...
* leave only the correct constant, either +43 ns or -43 ns
* please explain in the comment the source of the 43 ns (a reference to a document or talk is a good idea too); the existing reference only explains the 1.6 ms

Reply:
Thanks for the comment Gianluca, i agree what i left was confusing, i tried to add some documentation with some references, but i need to understand these offsets better so I hope to look into this a bit and add something that better explains the delays.